pkgs/development/libraries/libcanberra/default.nix

```diff
@@ -40,6 +40,11 @@ stdenv.mkDerivation rec {
       url = "https://raw.githubusercontent.com/macports/macports-ports/master/audio/libcanberra/files/patch-configure.diff";
       sha256 = "1f7h7ifpqvbfhqygn1b7klvwi80zmpv3538vbmq7ql7bkf1q8h31";
     }}
+  '' + lib.optionalString (stdenv.isDarwin && stdenv.isAarch64) ''
+    patch -p0 < ${fetchpatch {
+      url = "https://raw.githubusercontent.com/macports/macports-ports/5a7965dfea7727d1ceedee46c7b0ccee9cb23468/audio/libcanberra/files/dynamic_lookup-11.patch";
+      sha256 = "sha256-uuG+6PqzDKCzBt+pszN8Gu4yEDbUMzTZ+t/Hr5hzP9Q=";
+    }}
```

Comment:
Also, it is nicer to use URL like https://github.com/macports/macports-ports/raw/5a7965dfea7727d1ceedee46c7b0ccee9cb23468/audio/libcanberra/files/dynamic_lookup-11.patch since it makes it easier to get back to the repo.

Reply:
Done.